man/rgbasm-old.5

```diff
@@ -0,0 +1,342 @@
+'\" e
+.\"
+.\" SPDX-License-Identifier: MIT
+.\"
+.Dd October 21, 2024
+.Dt RGBASM-OLD 5
+.Os
+.Sh NAME
+.Nm rgbasm-old
+.Nd obsolete language documentation
+.Sh DESCRIPTION
+This is the list of features that have been removed from the
+.Xr rgbasm 5
+assembly language over its decades of evolution, along with their modern alternatives.
+Its goal is to be a reference for backwards incompatibility, when upgrading an old assembly codebase to work with the latest RGBDS release.
+It does
+.Em not
+attempt to list syntax bugs that were fixed, nor new reserved keywords that may conflict with old identifiers.
+.Sh REMOVED
+These are features which have been completely removed, without any direct alternatives.
```

Comment:
This is a bit confusing, they are "without any direct alternatives" but immediately after I see a "Instead, use explicit colons; for example,"

Reply:
Yeah, it feels like a blurry line between what's "removed" vs "replaced". My heuristic is "can a simple `sed` regex fix the syntax for you". So `ldio` gets replaced with `ldh`, `name: MACRO` gets replaced with `MACRO name`, etc. But if you were in the habit of using colon-less labels, a regex would have to replace `^([A-Za-z_][A-Za-z_0-9@#]*)` while ignoring all the keywords that could also match (e.g. column-1 `print 2+2` should not become `print:: 2+2`). So I count "colon-less labels" as "removed, you have to manually add colons".

Reply:
Looking over the list again, some other things should probably be swapped, e.g. `*` comments are `sed`-fixable.

Reply:
Okay, I've rearranged these.